src/reporter/github.js

```diff
@@ -53,6 +56,49 @@ export default class GitHub {
     }
   }
 
+  async loadAllIssues() {
+    try {
+      const issues = await this.octokit.paginate('GET /repos/{owner}/{repo}/issues', {
+        ...this.commonParams,
+        state: GitHub.ISSUE_STATE_ALL,
+        per_page: 100,
+      });
+
+      const onlyIssues = issues.filter(issue => !issue.pull_request); // Filter out pull requests since GitHub treats them as a special type of issue
+
+      onlyIssues.forEach(issue => {
+        const cachedIssue = this.issuesCache.get(issue.title);
+
+        if (!cachedIssue || new Date(issue.created_at) < new Date(cachedIssue.created_at)) { // Only cache the oldest issue if there are duplicates
+          this.issuesCache.set(issue.title, issue);
+        }
+      });
+
+      logger.info(`Cached ${onlyIssues.length} issues from the repository`);
+    } catch (error) {
+      logger.error(`Failed to load issues: ${error.message}`);
+    }
+  }
+
+  async refreshIssuesCache() {
```

Comment:
I would find it more straightforward if there was some `issues` getter that memoizes the promise and populates the cache if needed, and a `clearCache` method that does only that.